assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties

```diff
@@ -255,6 +255,9 @@ che.infra.docker.bootstrapper.installer_timeout_sec=180
 # Once servers for one installer available - checks stopped.
 che.infra.docker.bootstrapper.server_check_period_sec=3
 
+# URL rewriter. May be overriden in case of single-port mode e.t.c
+che.infra.docker.url_rewriter=default
+
```

Comment:
Make sure you are able to configure it with CLI

Reply:
Added